doc/python/cable_cell.rst

```diff
@@ -93,4 +93,147 @@ Cable cells
 
     properties of an ionic species.
 
+.. _pycableprobes:
+
+Cable cell probes
+-----------------
+
+Cable cell probe addresses are definied analagously to their counterparts in
+the C++ API (see :ref:`cable-cell-probes` for details). Sample data recorded
+by the Arbor simulation object is returned in the form of a NumPy array,
+with the first column holding sample times, and subsequent columns holding
+the corresponding scalar- or vector-valued sample.
+
+Location expressions will be realised as zero or more specific sites on
+a cell; probe addresses defined over location expressions will describe zero,
+one, or more probes, one per site.
+
+Each of the functions described below generates an opaque :class:`probe`
+object for use in the recipe :py:func:`get_probes` method.
+
+Membrane voltage
+   .. py:function:: cable_probe_membrane_voltage(locations)
+
+   Cell membrane potential (mV) at the sites specified by the location
+   expression string ``locations``. This value is spatially interpolated.
```

Comment:
It will work with labels; the expressions are evaluated in the back end with the corresponding cell description 'in hand', whose `mprovider` object is used to resolve locset expressions.

Reply:
I added a comment in the docs which I hope clarifies this.